lib/fast_jsonapi/serialization_core.rb

```diff
@@ -157,12 +156,13 @@ def get_included_records(record, includes_list, known_included_objects, params =
             included_objects = [included_objects] unless relationship_type == :has_many
 
             included_objects.each do |inc_obj|
+              serializer = (inc_obj.class.to_s + "Serializer").constantize if @relationships_to_serialize[item][:polymorphic].is_a?(Hash)
               if remaining_items(items)
                 serializer_records = serializer.get_included_records(inc_obj, remaining_items(items), known_included_objects)
                 included_records.concat(serializer_records) unless serializer_records.empty?
               end
 
-              code = "#{record_type}_#{inc_obj.id}"
+              code = "#{record_type}_#{inc_obj.class.to_s}_#{inc_obj.id}"
```

Comment:
the key needs to be thought through if we are going to support polymorphic relationships to be included

Reply:
Does Netflix have a naming standard? could it just be "inc_poly_obj"?

Reply:
@shishirmk Any updates here. I too would like to see this merged. As polymorphic includes are quite common